flytekit/common/constants.py

```diff
@@ -36,3 +36,9 @@ class SdkTaskType(object):
 class CloudProvider(object):
     AWS = "aws"
     GCP = "gcp"
+
+
+class DistributedTrainingContextKey(object):
+    CURRENT_HOST = "current_host"
+    HOSTS = "hosts"
+    NETWORK_INTERFACE_NAME = "network_interface_name"
```

Comment:
My thought was that this is for all the plugins that would use distributed training so that's why I didn't put in under `sagemaker` package. Does that not make sense?

Reply:
Moved